gc-azure-pipelines.yml

```diff
@@ -33,17 +33,24 @@ jobs:
 
   # TODO: Add.
   - job: GCNotebookValidation_Windows 
-      pool:
-        vmImage: windows-2019
+    pool:
+      vmImage: windows-2019
  
     steps:
     # Install dotnet.
     - task: UseDotNet@2
       displayName: Install .NET 8.0 
       inputs:
-        version: 8.x
+        version: 8.0.x
+    - task: UseDotNet@2
```

Comment:
Why do we install both 8.0 and 9.0 just to run the tests targetting 8.0?

Reply:
We need the NET 9.0 one because that's what's expected from the global.json to install the dependency of dotnet-repl. The projects are all 8.0 - we'll move these to .NET 9 once it's out but needed to get these to run unless anyone has better suggestions to fix this.

Reply:
ok - it is just unclear why from the context. A comment would be nice.